src/renderers/native/ReactNativeStackInspector.js

```diff
@@ -58,7 +60,9 @@ if (__DEV__) {
         measure: callback =>
           UIManager.measure(component.getHostNode(), callback),
         props: getHostProps(component),
-        source: component._currentElement && component._currentElement._source,
+        source: component &&
```

Comment:
I would probably prefer an early return of “empty hierarchy” rather than these kinds of checks.